scripts/spring19_seminar/cola_npi_x_sample.sh

```diff
@@ -0,0 +1,367 @@
+#!/bin/bash
+#SBATCH --time=12:00:00
+#SBATCH --mem=20000
+#SBATCH --gres=gpu:1
+#SBATCH --job-name=myrun
+#SBATCH --output=slurm_%j.out
+
```

Comment:
Add a comment here summarizing the naming scheme, and summarizing what combinations you're running.

Reply:
For the MNLI runs, it'll probably _much_ faster to train sequentially, since MNLI is pretty big:

- Run jiant to train a model on MNLI as a pretraining task.
- Then, for each CoLA/NPI config you're interested in, load the MNLI model (using a specified path), then pretrain on only the CoLA/NPI data. If it's not clear how to set this up, @sheng-fu or @zphang should be able to help.